packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -394,11 +401,26 @@ watch(
 	{ immediate: true }
 );
 
-function updateNodeModel(model: Model) {
-	if (model) {
-		amr.value = model;
-		createOutput(model);
-	}
+const isSaveDisabled = computed(() => {
+	const id = amr.value?.id;
+	if (!id) return true;
+	const projectAssets = useProjects().activeProject.value?.projectAssets;
+	const outputPort = props.node.outputs?.find((port) => port.value?.[0] === id);
+	return projectAssets?.some((asset) => asset.assetId === outputPort?.value?.[0]);
```

Comment:
I feel this could be a new method in the `useProject` composable.
Something like `withinActiveProjectAsset` or something better named, that could be used as:
```js
return useProjects().withinActiveProjectAsset(outputPort?.value?.[0]);
```